test/tests/ghost-basics/run.sh

```diff
@@ -27,4 +27,10 @@ _request() {
 . "$dir/../../retry.sh" '_request GET / --output /dev/null'
 
 # Check that /ghost/ redirects to setup (the image is unconfigured by default)
-_request GET '/ghost/' -I | grep -q '^Location: .*setup'
+if [[ $serverImage == ghost:1* ]]
+then
+	_request GET '/ghost/api/v0.1/authentication/setup/' | grep -q 'status":false'
```

Comment:
How often will the api version change?  It'll be a bit annoying if failing tests just mean the test needs a bump (like we have now with 1.0).  Maybe we just need to do a simpler test to see that a `GET` request succeeds?

 ```bash
_request GET '/ghost/' -L --output /dev/null
```